src/store/modules/picklist/mutations.ts

```diff
@@ -3,11 +3,14 @@ import PicklistState from './PicklistState'
 import * as types from './mutation-types'
 
 const mutations: MutationTree <PicklistState> = {
-  [types.PICKLIST_CURRENT_UPDATED] (state, payload) {
+  [types.PICKLIST_CURRENT] (state, payload) {
```

Comment:
Improve mutations names. It should have information about the actions being done

Reply:
Done